### DISCUSSION TITLE
Feature requese:  Introducing a Second Discord Console Channel ID for Enhanced Log Management

### DISCUSSION THREAD

**Discussion by CtrlAiDel** [DiscordSRV/DiscordSRV#1681] [f. Questions] [answered]
Hi guys

My request concerns adding a second type of DiscordConsoleChannelId. The first type remains the same, allowing owners and operators (OPs) to type commands in the console as OPs.

The second type would be dedicated to logs only, named DiscordLogChannelId. This channel would function similarly to the first but would be protected from user input. The user would configure this via Discord permissions, ensuring only users with the appropriate permissions could view the logs while regular users would not be able to type commands. This setup could be explained in the commentary of the command and would enable admins to supervise each other and monitor for crashes effectively.

**Comment by underscore11code** [comment]
What would a second channel accomplish that couldn't be done by setting the existing console channel to be viewable by everyone? Read and write permissions on Discord are separate for a reason.

**Reply by CtrlAiDel** [reply]
I would give the read permission only to the admin * Sorry I haven't told you that. 

It's for logs, so it's for admin, but no access to the console

**Reply by CtrlAiDel** [reply]
So to be be more precise if its needed.

DiscordConsoleChannelId: Viewable and writable via the owner and op
DiscordLogChannelId: Only viewable via the admins with the needed perms. They could not write in the channel off course.

I've looked everywhere for a discord bot that copy one channel to another with much success. I have done that for an hour before contacting you guys. 

Thanks good evening / night!

**Reply by underscore11code** [reply]
Again, I understand what you're asking, but I don't understand why you need it? What does splitting the console into a channel that can run commands + displays logs, and a console that *only* displays logs do for you that can't be accomplished by giving read access to the admin group? Everything in the current console channel would be forwarded to this hypothetical log channel, so it's not like you're hiding stuff in the console channel.

**Reply by CtrlAiDel** [reply]
The goal is to achieve the best of both worlds: both the OP and owner having a console RCON channel for typing commands on 
the go.

And the admin being able to read and supervise without having access to OP commands.

**Reply by granny** [reply, marked as answer]
That is possible to do with one channel. Just make it so Admins don't have the send messages perm in the console channel.

**Comment by CtrlAiDel** [comment]
I get it now im sorry for the waste of time


Thanks granny i miss you! 💙🙃
